subsystem/infra_env_test.go

```diff
@@ -358,7 +358,8 @@ var _ = Describe("Infra_Env", func() {
 		}
 		Expect(contents).To(ContainSubstring("eth0"))
 	},
-		Entry("ocp versions greater than/ equal to 4.14", staticnetworkconfig.MinimalVersionForNmstatectl),
+		Entry("ocp versions greater than/ equal to 4.14, x86 arch", staticnetworkconfig.MinimalVersionForNmstatectl, common.X86CPUArchitecture),
+		Entry("ocp versions greater than/ equal to 4.14, arm arch", staticnetworkconfig.MinimalVersionForNmstatectl, common.ARM64CPUArchitecture),
 		Entry("ocp versions less than 4.14", "4.12"),
```

Comment:
Seems 'edge-subsystem-aws ' [job](https://prow.ci.openshift.org/view/gs/test-platform-results/pr-logs/pull/openshift_assisted-service/6798/pull-ci-openshift-assisted-service-master-edge-subsystem-aws/1838620676877979648) fails due the a missing 'arch' arg here.